src/hooks/useDeleteMeeting.ts

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from "react";
+import has from "lodash/has";
```

Comment:
done

Reply:
Caught the fail because I'm using a lot of functions from `lodash`. And there is a conflict in the names of the functions inside the different modules and `Vite` doesn't know what to do with it. 

![image](https://user-images.githubusercontent.com/11043120/231135225-5da47b91-314e-4ebb-b551-3a0b32bb5a0c.png)